pkg/controller/node/status.go

```diff
@@ -167,9 +167,8 @@ func isNodeMCDFailing(node *corev1.Node) bool {
 		isNodeMCDState(node, daemonconsts.MachineConfigDaemonStateUnreconcilable)
 }
 
-// getUpdatedMachines filters the provided nodes to ones whose current config
-// matches the desired config, which also matches the target config.
-// and the "done" flag is set.
```

Comment:
this is actully true if you look at `isNodeDoneAt`

Reply:
I see. I thought having "desired config" and "target config" at the same time was confusing.